yourls/.env-dist

```diff
@@ -0,0 +1,58 @@
+# The docker image to use:
+YOURLS_IMAGE=yourls:1.9.2
+
+# The domain name for the yourls service:
+YOURLS_TRAEFIK_HOST=yourls.example.com
+
+# The name of this instance. If there is only one instance, use 'default'.
+YOURLS_INSTANCE=
+
+# Filter access by IP address source range (CIDR):
+##Disallow all access: 0.0.0.0/32
+##Allow all access: 0.0.0.0/0
+YOURLS_IP_SOURCERANGE=0.0.0.0/0
+
+# HTTP Basic Authentication:
+# Use `make config` to fill this in properly, or set this to blank to disable.
+YOURLS_HTTP_AUTH=
+
+# OAUTH2
+# Set to `true` to use OpenID/OAuth2 authentication via the
+# traefik-forward-auth service in d.rymcg.tech.
+# Using OpenID/OAuth2 will require login to access your app,
+# but it will not affect what a successfully logged-in person can do in your
+# app. If your app has built-in authentication and can check the user
+# header that traefik-forward-auth sends, then your app can limit what the
+# logged-in person can do in the app. But if your app can't check the user
+# header, or if your app doesn't have built-in authentication at all, then
+# any person with an account on your Gitea server can log into your app and
+# have full access.
+YOURLS_OAUTH2=
+# In addition to Oauth2 authentication, you can configure basic authorization
+# by entering which authorization group can log into your app. You create
+# groups of email addresses in the `traefik` folder by running `make groups`. 
+YOURLS_OAUTH2_AUTHORIZED_GROUP=
+
+# Mutual TLS (mTLS):
+# Set true or false. If true, all clients must present a certificate signed by Step-CA:
+YOURLS_MTLS_AUTH=false
+# Enter a comma separated list of client domains allowed to connect via mTLS.
+# Wildcards are allowed and encouraged on a per-app basis:
+YOURLS_MTLS_AUTHORIZED_CERTS=*.clients.yourls.example.com
+
+YOURLS_PROXY=10.13.16.1
```

Comment:
Please add comments describing what this is.

Reply:
Does it work if you're not using the VPN?

Reply:
[This](https://yourls.org/docs/guide/advanced/proxy#proxy-with-authentication) explains what YOURLS_PROXY is for (well, barely). My assumption was that without this var, YOURLS wouldn't function correctly or at all. I haven't tested without YOURLS_PROXY.

Reply:
The way I read that, it means to use this as an outgoing proxy, which you don't need.

Reply:
Ah, OK. I removed `YOURLS_PROXY` in [de78ef4](https://github.com/EnigmaCurry/d.rymcg.tech/pull/280/commits/de78ef4b69ba5106d900bc0332b3f7e8363bc696)